plugin/handlergen.go

```diff
@@ -294,22 +290,22 @@ func (p *OrmPlugin) removeChildAssociations(message *generator.Descriptor) bool
 		// Prep the filter for the child objects of this type
 		keys := p.findAssociationKeys(message, typeNames[typeName], field)
 		childFKeyTypeName := ""
-		_, _, fieldTypeNameORM := getTypeNames(typeNames[rawFieldType])
-		p.P(`filterObj`, rawFieldType, ` := `, fieldTypeNameORM, `{}`)
+		fieldTypeName := p.TypeName(typeNames[rawFieldType])
+		p.P(`filterObj`, rawFieldType, ` := `, fieldTypeName, `ORM{}`)
 		for k, v := range keys {
 			for _, childField := range typeNames[rawFieldType].GetField() {
-				if strings.EqualFold(childField.GetName(), k) {
+				if strings.EqualFold(generator.CamelCase(childField.GetName()), k) {
```

Comment:
what's the purpose of `CamelCase` if you're comparing ignoring case? I'm guessing it translates snake case?

Reply:
Exactly. This should actually be the fix for issue #28 for cases where the ID fields are defined in snake case, camel case, camel case with initialisms, or even StUdLyCaPs.